components/graph/graph-zoom.directive.ts

```diff
@@ -30,7 +31,7 @@ import { NzZoomTransform, RelativePositionInfo } from './interface';
   standalone: true
 })
 export class NzGraphZoomDirective implements OnDestroy, AfterViewInit {
-  @Input() nzZoom?: number;
+  @Input({ transform: numberAttributeWithZeroFallback }) nzZoom?: number;
```

Comment:
Since nzMinZoom defaults to `0.1`, so I don't think it's a good idea that the fallback value of nzZoom is `0`, WDYT